extension/modules/service.js

```diff
@@ -1363,7 +1363,7 @@ var PersonaService = {
         if (uri.host != this.addonsHost)
             return;
 
-        if (endsWith(uri.filePath, "/users/logout") || aRequest.requestMethod == "POST" && endsWith(uri.filePath, "/users/login"))
+        if (endsWith(uri.filePath, "/users/logout") || aRequest.requestMethod == "POST" && endsWith(uri.filePath, "/users/login") || endsWith(uri.filePath, "/api/v3/accounts/authenticate/"))
```

Comment:
I was talking about the url this patch adds. We only need ´POST` requests for that. Refreshing the favorites does not make sense if the request is a `GET` for example, as that means the login page has just loaded but not been submitted.